support-frontend/test/admin/settings/ContributionTypesSpec.scala

```diff
@@ -0,0 +1,20 @@
+package admin.settings
+
+import admin.settings.AmountsTests.AmountsTests
+import io.circe
+import io.circe.parser.decode
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.io.Source
+
+class ContributionTypesSpec extends AnyWordSpec with Matchers {
+  "ContributionTypes" should {
+    "decode correctly" in {
+      val json: String = Source.fromURL(getClass.getResource("/contributionTypes.json")).mkString
+      val contributionTypes: Either[circe.Error, ContributionTypes] = decode[ContributionTypes](json)
+      contributionTypes.isRight mustBe true
+      contributionTypes.map(contributionTypes => contributionTypes.GBPCountries.size mustBe 3)
```

Comment:
this makes it much nicer
https://www.scalatest.org/user_guide/using_EitherValues
```suggestion
class ContributionTypesSpec extends AnyWordSpec with Matchers with EitherValues {
  "ContributionTypes" should {
    "decode correctly" in {
      val json: String = Source.fromURL(getClass.getResource("/contributionTypes.json")).mkString
      val contributionTypes: Either[circe.Error, ContributionTypes] = decode[ContributionTypes](json)
      contributionTypes.right.value.GBPCountries.size mustBe 3
```